traffic_ops/testing/api/v3/cdns_test.go

```diff
@@ -92,6 +93,16 @@ func CreateTestCDNs(t *testing.T) {
 
 }
 
+func SortTestCDNs(t *testing.T) {
+	var header http.Header
+	for _, cdn := range testData.CDNs {
+		_, reqInf, err := TOSession.GetCDNByNameWithHdr(cdn.Name, header)
```

Comment:
So this doesn't really verify that the response is sorted alphabetically. Maybe we should take the response of all the CDNs, iterate over them, and use `strings.Compare` to make sure the response is sorted as expected.

Reply:
Fixed test case.